lib/checkbufferoverrun.cpp

```diff
@@ -881,6 +880,8 @@ std::string CheckBufferOverrun::MyFileInfo::toString() const
 
 bool CheckBufferOverrun::isCtuUnsafeBufferUsage(const Check *check, const Token *argtok, MathLib::bigint *offset, int type)
 {
+    if (!offset)
```

Comment:
Moved this check here since it would do all the work and then bailout if no output parameter was provided.